app/models/ansible_role.rb

```diff
@@ -1,6 +1,10 @@
 # Simple model to store basic info about the Ansible role
 class AnsibleRole < ActiveRecord::Base
+  include Authorizable
+
+  self.include_root_in_json = false
```

Comment:
The importing form posts individual roles as json, this is for the sake of having a simple array of js objects where each object is a role.